### PR TITLE
Separate ECMA-262 and ECMA-402 parts of calendar- and locale-dependent spec text

### PR DESCRIPTION
**Pull request by ptomato**
Closes: #1243

### CODE REVIEW

File: spec.html [hunk 1]
```diff
@@ -25,6 +25,14 @@ <h1>Introduction</h1>
   <emu-figure id="temporal-persistence-model" caption="Temporal String Persistence">
     <object data="docs/persistence-model.svg" type="image/svg+xml"></object>
   </emu-figure>
+
+  <h2>How to Read</h2>
```
[the remark this answers is not in the document]
Comment (ptomato) [reply spec.html:29]: That produces odd results, in that the How to Read clause is numbered "1" but doesn't show up in the navigation pane on the left, so the numbering there weirdly starts at "2". If we can't use `<h2>` then I'll just delete the heading and leave this as a paragraph in the introduction.

File: spec/intl.html [hunk 2]
```diff
@@ -805,4 +805,131 @@ <h1><a href="https://tc39.es/ecma402/#sec-properties-of-intl-datetimeformat-inst
       Finally, Intl.DateTimeFormat instances have a [[BoundFormat]] internal slot that caches the function returned by the format accessor (<emu-xref href="#sec-intl.datetimeformat.prototype.format"></emu-xref>).
     </p>
   </emu-clause>
+
+  <emu-clause id="locale-sensitive-functions">
+    <h1><a href="https://tc39.es/ecma402/#locale-sensitive-functions">Locale Sensitive Functions of the ECMAScript Language Specification</a></h1>
+
+    <ins class="block">
+      <emu-clause id="sup-properties-of-the-temporal-duration-prototype-object">
```
[the remark this answers is not in the document]
Comment (ptomato) [reply spec/intl.html:1200]: I copied it from the way ECMA-402 does this: https://github.com/tc39/ecma402/blob/master/spec/locale-sensitive-functions.html#L12
I assume it stands for "supersedes" or something like that.

File: spec.html (hunk 1 above)
[the remark this answers is not in the document]
Comment (Ms2ger) [reply spec.html:29]: Oh, that's annoying. I think it's fine to just drop the header.

File: spec/duration.html [hunk 3]
```diff
@@ -533,19 +533,17 @@ <h1>Temporal.Duration.prototype.toJSON ( )</h1>
 
     <emu-clause id="sec-temporal.duration.prototype.tolocalestring">
       <h1>Temporal.Duration.prototype.toLocaleString ( [ _locales_ [ , _options_ ] ] )</h1>
+      <p>
+        An ECMAScript implementation that includes the ECMA-402 Internationalization API must implement the `Temporal.Duration.prototype.toLocaleString` method as specified in the ECMA-402 specification. If an ECMAScript implementation does not include the ECMA-402 API the following specification of the `toLocaleString` method is used.
```
[the remark this answers is not in the document]
Comment (ptomato) [reply spec/duration.html:537]: I'm not confident at all about this, but my understanding was that the idea was to decouple Temporal Stage 3 from Intl.DurationFormat Stage 3, and just not have an "override" in the 402 section until Intl.DurationFormat progresses... @littledan?

File: spec/intl.html [hunk 4]
```diff
@@ -810,6 +810,393 @@ <h1><a href="https://tc39.es/ecma402/#sec-properties-of-intl-datetimeformat-inst
...
+            1. If _calendar_.[[Identifier]] is *"iso8601"*, then
+              1. Let _overflow_ be ? ToTemporalOverflow(_options_).
+              1. Set _fields_ to ? ToTemporalDateFields(_fields_, « *"day"*, *"month"*, *"year"* »).
+              1. Let _year_ be ? Get(_fields_, *"year"*).
+              1. Let _month_ be ? Get(_fields_, *"month"*).
+              1. Let _day_ be ? Get(_fields_, *"day"*).
+              1. Let _result_ be ? RegulateDate(_year_, _month_, _day_, _overflow_).
+              1. Set _year_ to _result_.[[Year]].
+              1. Set _month_ to _result_.[[Month]].
+              1. Set _day_ to _result_.[[Day]].
```
[the remark this answers is not in the document]
Comment (ptomato) [reply spec/intl.html]: OK, hopefully this is more concise.

**Review by Ms2ger — APPROVED** [review]

**Pull request merged**